cpp/ql/src/semmle/code/cpp/models/implementations/StdContainer.qll

```diff
@@ -50,7 +88,13 @@ private class StdSequenceContainerConstructor extends Constructor, TaintFunction
  * The standard container function `data`.
  */
 private class StdSequenceContainerData extends TaintFunction {
-  StdSequenceContainerData() { this.hasQualifiedName("std", ["array", "vector"], "data") }
+  StdSequenceContainerData() {
```

Comment:
I think you want to use the `this.getClassAndName` trick to get a good join-order similar to what we do [here](https://github.com/github/codeql/blob/main/cpp/ql/src/semmle/code/cpp/models/implementations/StdString.qll#L72).